packages/textInput/components/TextInputWithButtons.tsx

```diff
@@ -49,13 +54,17 @@ const TextInputWithButtons = (props: TextInputWithButtonsProps) => {
       return "disabled";
     }
     if (hasFocus) {
-      return `${props.appearance}-focus`;
+      return `${appearance}-focus`;
     }
-    return props.appearance;
+    return appearance;
   };
 
   const getInputElementProps = () => {
-    const baseProps = getBaseInputElementProps(props);
+    const baseProps = getBaseInputElementProps({
+      type,
+      appearance,
+      showInputLabel
```

Comment:
I believe this is missing the placeholder props. Not sure if it missing anything else like hint content.

Reply:
Ah yep, this should be resolved now 👍